src/metabase/api/collection.clj

```diff
@@ -733,15 +767,15 @@
     :snippet    :model/NativeQuerySnippet
     :timeline   :model/Timeline))
 
-(defn- post-process-rows
+(defn post-process-rows
   "Post process any data. Have a chance to process all of the same type at once using
   `post-process-collection-children`. Must respect the order passed in."
-  [collection rows]
+  [options collection rows]
```

Comment:
Same question about passing options first, is there a good reason not to pass it last like usual?

Reply:
Same answer as above although less tied to this since it's only used once and it's not in a threading form.